source/variables/breakpoints.css

```diff
@@ -0,0 +1,13 @@
+/* 
+The breakpoints for all resolutions represented as pairs with the breakpoint name and its associated pixel width.
+http://cssnext.io/features/#custom-media-queries 
+@media (--medium) { ..styles.. }
+*/
+:root {
+	@custom-media --breakpoint--extra-small (width <= 320px) and (width >= 0); /* Used for style that should not cascade out */
```

Comment:
Ah right, we don't have that em() functionality anymore to calculate a pixel value off base font size. 
```
@function em($target, $options: ()) {
	$options: map-merge((
		context: $baseFontSize
	), $options);

	$output: null;
	$context: map-get($options, context);

	@if type-of($target) == list {
		$output: unquote('') !default;

		@each $value in $target {
			$value: em($value, (context: $context));

			$output: append($output, $value);
		}
	}
	@else {
		$target: parseInt($target);
		$context: parseInt($context);

		$output: ($target / $context) * 1em;
	}

	@return $output;
}
```
That's not to say I didn't try though: https://github.com/andyjansson/postcss-functions/issues/17
You can build your own functions for postcss, however I was not able to get it to pass a pixel value and return a number, nor how to obtain our CSS baseline font value in the JS side of things.

So these will have to be set to actual ems, but I don't think that will effect anything.